docker/mu-plugins/jetpack-debug-helper/class-admin.php

```diff
@@ -87,10 +87,10 @@ public function render_ui() {
 	 * Store options.
 	 */
 	public function update_option() {
-
+		check_admin_referer( 'store-debug-modules' );
```

Comment:
I had to move this up here to pass phpcs checks. How did this pass before?

Reply:
Not sure. But this doesn't work because the check will fail when there is no form posted. Looking at it